src/viking.js

```diff
@@ -28,11 +28,11 @@ class Soldier {
       if(this.health <= 0 ){ 
           return `${this.name} has died in act of combat`;
      } else {
-         return `${this.name} has received ${Damage} points of damage`;
+         return `${this.name}has received ${Damage} points of damage`;
```

Comment:
failing this test "should return "NAME has received DAMAGE points of damage", if the Viking is still alive" because you forgot to put a space between "this.name" and "has":
```
return `${this.name} has received ${Damage} points of damage`;
```